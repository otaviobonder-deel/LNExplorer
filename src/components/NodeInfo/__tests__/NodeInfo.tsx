import React from "react";
import renderer from "react-test-renderer";
import { NodeInfo } from "../index";

it("should render each node info", () => {
  const info = {
    publicKey: "test123",
    alias: "test",
    color: "#fff",
    visible: true,
    links: [
      {
        channelId: "123",
        node1: "test123",
        node2: "test345",
        capacity: "10000",
        color: "#fff",
      },
    ],
  };
  const graph2ScreenCoords = () => ({ x: 0, y: 0, z: 0 });
  const tree = renderer
    .create(<NodeInfo graph2ScreenCoords={graph2ScreenCoords} info={info} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
