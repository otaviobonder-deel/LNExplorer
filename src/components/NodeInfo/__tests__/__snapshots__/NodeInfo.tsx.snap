// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render each node info 1`] = `
<div
  className="MuiPaper-root makeStyles-container-1 MuiPaper-elevation1 MuiPaper-rounded"
  style={
    Object {
      "left": 0,
      "top": 20,
    }
  }
>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Pubkey: 
    test123
  </p>
  <p
    className="MuiTypography-root MuiTypography-body1"
  >
    Alias: 
    test
  </p>
</div>
`;
